docker/docker-compose.yaml

```diff
@@ -164,6 +141,44 @@ services:
     labels:
       - traefik.enable=false
 
+  # Web Admin tools to monitor services
+  # Celery Flower
+  celery-flower:
+    image: unstract/backend:${VERSION}
+    container_name: unstract-celery-flower
+    restart: unless-stopped
+    entrypoint: .venv/bin/celery
+    command: "-A backend flower --port=5555 --purge_offline_workers=5"
+    env_file:
+      - ../backend/.env
+    depends_on:
+      - execution-consumer
+      - redis
+    labels:
+      - traefik.enable=false
+    ports:
+      - "5555:5555"
+    environment:
+      - ENVIRONMENT=development
+    volumes:
+      - unstract_data:/data
+    profiles:
+      - optional
+
+  # Cerebro
+  cerebro:
```

Comment:
@chandrasekharan-zipstack no need to add cerebro part of docker compose. we can plug it for debugging purpose if required.

Reply:
Okay, shall I mark it with optional profile so that it can be used for debugging similar to flower? This is already under the profile `unstract-metrics` by the way.

If its still not needed, I'll add a TODO to remove it once my development is done